libraries/ssh_crypto.rb

```diff
@@ -48,6 +48,13 @@ def valid_ciphers
       when /7\./
         ciphers = ciphers66
       end
+    when 'fedora'
+      ciphers = ciphers66
+    when 'opensuse'
+      case inspec.os[:release]
+      when /13\.2/
+        ciphers = ciphers66
+      end
```

Comment:
good catch, however this is a generic problem here. I'll fix it for all platforms